### ISSUE TITLE
Login screen issues after upgrading from v3.5.15.1 to v3.6.0.1

### ISSUE THREAD

**Issue by sidm1983** [craftcms/cms#7450]
### Description
The CP login screen looks cramped and has minor issue after updating to v3.6.0.1

### Steps to reproduce

1. Install or update to v3.6.0.1
2. Go to the CP login screen
3. You will see that the following issues:
   - Login screen looks heaps more cramped than the login screen on v3.5.15.1
   - You can see that the "forgot your password?" link is right above the "Keep me logged in" checkbox without much space in between the two.
   - The site name at the top of the login form has also shrunk in size compared to v3.5.15.1.
   - Lastly, the "Login" button stays disabled even though a user name and password has been entered. The login button itself works when clicked, but the "disabled" class on it doesn't get removed, so still looks unclickable.

### Additional info

- Craft version: 3.6.0.1
- PHP version: 7.3
- Database driver & version: PGSQL 11
- Plugins & versions: NA

**Comment by brandonkelly** [comment]
The login screen hasn’t had any styling changes in 3.6.

Sounds like the new CSS and/or JavaScript isn’t making its way to your browser. Could be a couple reasons for that:

- The update didn’t get installed in full
- Somehow Craft missed that there was updated CSS/JS
- Your `web/cpresources/` folder isn’t writable

Try deleting all contents of your `web/cpresources/` folder as a starting point, and then force-refresh the page. If that works, great. Otherwise you will probably see that the page is now completely unstyled, which would indicate that Craft isn’t able to write to the `web/cpresources/` folder anymore – so a permissions issue.

**Comment by sidm1983** [comment]
Hi @brandonkelly,

It is definitely not a permission issue on the `web/cpresources/` folder as the various folders/files within that get regenerated every time I delete the contents of the folder and refresh the CP login page. Also, when I ssh into my docker container that runs Craft CMS, I can see that the owner of the folder is the web user and the folder and all its descendants have a permission of 777.

I have also tried emptying out my `storage` folder as well as my `vendor` folder and doing a `composer install` again, but that doesn't help.

Also, since I am running this in docker, I also tried doing a `docker-compose down` and removing all my running containers, networks and their volumes so that I could do a fresh `docker-compose up` command and have everything recreated from scratch and I still get the cramped looking login screen with all the issues I have mentioned above.

I have also checked the `web.log` file and can't see any errors there. Nothing in the browser console either.

I can however see that the CSS seems to be styling the page in that way and that is what led me to think that perhaps something in the CSS broke between 3.5.15.1 & 3.6.0.1. However, since you said that the login screen has not had any styling changes in 3.6, I am running out of ideas on what else to check in order to troubleshoot the issue.

I will try to switch back to another branch that is still running version 3.5.15.1 and see if that fixes the issue. Will let you know how I go. In the meantime, any other thoughts or pointers would be greatly appreciated.

Thank you.

**Comment by sidm1983** [comment]
@brandonkelly, so I went back to v3.5.15.1 and compared the CSS to v3.6.0.1 and I can definitely see differences. In the file `login.css`, I see the following difference (which explains why the heading size has shrunk):
- login.css in v3.5.15.1 has `#login-form h1 { text-align: center; font-size: 3em; color: #3f4d5a; word-wrap: break-word; }`
- login.css in v3.6.0.1 has `#login h1 { text-align: center; font-size: 3em; word-wrap: break-word; }`

However, the id of the form is actually `login-form` which means that the CSS in v3.6.0.1 is not targeting the `h1` element correctly and therefore it is not getting set to `font-size: 3em`.

Any thoughts on this?

**Comment by brandonkelly** [comment]
It’s meant to target this div:

https://github.com/craftcms/cms/blob/81cba0ca171cfad5996470dcd8ba6087b96a91bf/src/templates/login.html#L33

So that opens another possibility, which is that Twig didn’t recompile the new login template for whatever reason.

Try clearing out your `storage/runtime/compiled_templates/` folder, or run the following command:

```bash
> php craft clear-caches/compiled-templates
```

**Comment by sidm1983** [comment]
Hi @brandonkelly, I have tried clearing out my entire storage folder including the runtime folder within it and it still doesn't seem to work. I also have a startup script that runs the clear cache command for the compiled templates during the startup of my Craft CMS docker container. That still didn't fix it.

Will try and investigate further and look for that div with an id of "login". I don't remember seeing that in the markup when I inspected it in my browser's dev tools.

**Comment by brandonkelly** [comment]
If even that didn’t work, then maybe Composer just never installed Craft 3.6 in full. Try nuking your whole `vendor/` folder and then running `composer update`.

**Comment by sidm1983** [comment]
@brandonkelly, I have already tried nuking the `vendor` folder previously as well as my `composer.lock` file and have tried the `composer update` command as well, but I am still getting the issue. This one has got me stumped at the moment. 🤔

**Comment by sidm1983** [comment]
@brandonkelly, so I can see the new `login.html` template file that you referenced above in my `vendor/craftcms/cms/src/templates` folder, but the login screen still somehow rendering the old markup. 🤯

I have tried to rebuild my docker image from scratch and did so after removing/clearing out the following files/folders:
- vendor
- storage
- web/cpresources/
- composer.lock

Could this be a composer version issue? I can see in the craft cms `composer.json` file that it requires composer version 2.0.8, but when my docker image is being built, it downloads the latest which is version 2.0.9. I wouldn't have thought this would cause any issues. Any thoughts on this?

**Comment by sidm1983** [comment]
@brandonkelly, I think I have figure out what is generally causing the issue, but I haven't pinpointed what specifically is the cause.

It looks like the issue is being caused by something in my project config yaml files. To test, I completely removed my project config folder and rebuilt my Craft CMS docker image and now the login form looks right.

I am going to continue to investigate to see if I can pinpoint which yaml file is causing the issue, but is there anything you can think of within the project config that could cause this kind of issue with the login form?

**Comment by sidm1983** [comment]
@brandonkelly, I finally figured out the exact reason for the issue! 😭

The login page was being hijacked by Flipbox Factory's SAML Service Provider plugin which I had installed. That plugin has a copy of the Craft CMS login page markup within its template files and it was replacing the latest login.html from Craft CMS v3.6 with older markup.

I will raise an issue with flipbox factory about this. Apologies for thinking it was an issue with Craft CMS. Thank you for your patience and help with troubleshooting 🙌🏽

I am going to close this issue now.
